module/VuFind/src/VuFind/Sitemap/Generator.php

```diff
@@ -374,7 +374,7 @@ protected function buildIndex($sitemaps)
                 }
             }
 
-            foreach ($sitemaps ?? [] as $sitemap) {
+            foreach ($sitemaps as $sitemap) {
```

Comment:
Should we add a typehint to the argument above, to ensure this is really true?

Reply:
I agree, done